lib/core/app-config/app-config.service.ts

```diff
@@ -34,7 +36,16 @@ export class AppConfigService {
         alfrescoRepositoryName: 'alfresco-1'
     };
 
+    private onChangeSubject: BehaviorSubject<any>;
+    onChange: Observable<any>;
+
     constructor(private http: HttpClient) {
+        this.onChangeSubject = new BehaviorSubject(this.config);
+        this.onChange = this.onChangeSubject.asObservable();
+    }
+
+    onLoad(): Observable<any> {
+        return this.onChange.map((config) => config).distinctUntilChanged();
```

Comment:
We will subscribe to the onChange observable so i don't think we need it. Maybe this could have sense if we stream  here the properties of the app config instead of the whole thing